app/renderer/components/preferences/payment/disabledContent.js

```diff
@@ -55,67 +55,66 @@ const styles = StyleSheet.create({
     marginTop: globalStyles.spacing.panelMargin
   },
 
-  paymentsMessage: {
+  disabledContent__message: {
     backgroundColor: globalStyles.color.lightGray,
     borderRadius: globalStyles.radius.borderRadiusUIbox,
+    boxSizing: 'border-box',
     padding: '40px',
     fontSize: paymentStyles.font.regular,
     lineHeight: '1.8em',
-    color: globalStyles.color.mediumGray,
-    width: '500px'
+    color: globalStyles.color.mediumGray
   },
 
-  text: {
+  disabledContent__message__header: {
+    fontSize: '18px',
+    paddingBottom: '0.5em'
+  },
+
+  disabledContent__message__text: {
     padding: '0.5em 0'
   },
 
-  textSide: {
-    fontSize: paymentStyles.font.regular,
-    margin: '50px 0 20px 12px'
+  disabledContent__message__text_bold: {
+    fontWeight: 'bold'
   },
 
-  h3: {
-    fontSize: '18px',
-    paddingBottom: '0.5em'
+  disabledContent__sidebar: {
+    minWidth: '200px',
+    marginLeft: '35px'
   },
 
-  sideH2: {
+  disabledContent__sidebar__header: {
+    opacity: 0.8,
     fontSize: '18px',
-    margin: '70px 0 -15px 12px'
+    margin: '70px 0 30px'
   },
 
-  boldText: {
-    fontWeight: 'bold'
+  disabledContent__sidebar__text: {
+    opacity: 0.8,
+    fontSize: paymentStyles.font.regular
   },
 
-  paymentsSidebar: {
-    opacity: 0.8,
```

Comment:
this one was removed, applying it to `disabledContent__sidebar__header ` and `disabledContent__sidebar__text`, to display the logos without opacity.